index.html

```diff
@@ -5,11 +5,26 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<title>Découverte de l'Open Source</title>
 		<link rel="stylesheet" href="/style.css">
+		<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400&display=swap" rel="stylesheet">
```

Comment:
Can be replaced with a single link (`https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap`) to reduce the number of network requests